src/seeds.bioregion.cpp

```diff
@@ -160,15 +165,33 @@ ACTION bioregion::join(name bioregion, name account) {
     auto mitr = members.find(account.value);
 
     if (mitr != members.end()) {
-        check(false, "user exists TODO allow users to switch");
-    } else {
-        members.emplace(_self, [&](auto & item) {
-            item.bioregion = bioregion;
+        if (mitr -> bioregion == bioregion) { return; }
+        leave(mitr -> bioregion, account);
```

Comment:
Join should still fail - the user needs to explicitly leave the bioregion

It doesn't make sense to "leave" automatically, but then fail, because they can't yet join due to the delay?!

So what I mean to say is - this join will fail in this case as leaving and immediately joining won't work, so might as well just fail here, instead of "leave" - same result.

Reply:
Yes, you are right. I fixed that